### ISSUE TITLE
Spread items across columns

### ISSUE THREAD

**Issue by Amorymeltzer** [emacs-dashboard/emacs-dashboard#302]
Semi-related to #111, but it'd be neat to be able to spread long lists over multiple columns.

**Comment by jcs090218** [comment]
It would be great if you have some references so we are able to know what exactly you are referring to! Thanks! :)

**Comment by ddoherty03** [comment]
I upvote this as well, similar to #111.

**Comment by Amorymeltzer** [comment]
Yeah, sorry @jcs090218 for this incredibly unhelpful issue; I struggled for words to describe it but I know how annoying that can be.

Basically, it's not that hard, especially on a laptop, for 5-10 items in 3-4 sections to take up most of the vertical space, while leaving a lot of horizontal space unused.  YMMV, but folks might appreciate a way to take a single long list — say, 18 recent files — and tile them in a three-column list of six items each.  I figure if #111 ever happened this would be doable there, but I opened it separate from #111 since it's more about columns than defining a grid system.  Truth be told, that's probably a requirement in order to keep everything delineated.

Hopefully that makes more sense?  Again, sorry for the bland note.

**Comment by ricardoricho** [comment]
Hi, I created https://github.com/ricardoricho/dashboard-columns, is more a hack than a package because it overwrites
many functions of the original dashboard. 
There are many issues, to name some:
At this moment recent files are not supported due to the core function  `dashboard-insert-startupify-lists`, it always shows icons (doesn't check if they are supported), the projects and bookmarks formats are fixed, and it truncates items always at the end. 
As always, PR and report issues are welcome.
To manage the items in a faster way, I suggest use of [ace-link-dashboard.](https://github.com/emacs-dashboard/ace-link-dashboard)
